### DISCUSSION TITLE
Face Swap Option not working

### DISCUSSION THREAD

**Discussion by simplymaya** [lllyasviel/Fooocus#2821] [Q&A]
Initially when I started using Fooocus, the face swap was generating images, but when I try now, the tool stops responding and Google colab also does not give any error. I thought it will take time but even after hours, nothing. Anyone else facing a similar issue or knows what can be done to keep it going?

**Comment by mashb1t** [comment]
Google Colab offers limited hardware specifications, which means that you can't use all features there (see readme, Colab section).
I assume the process just freezes and doesn't break or exit, right? Can you please check the resources inColab using the hardware monitor and upload a screenshot of it while generating with faceswap?

**Reply by simplymaya** [reply]
You are right, it just freezes, however, it was working when I started using it, but after a day or two, it stopped. I will share a snippet as soon as I get some time.
